src/System.Net.Http/tests/FunctionalTests/HttpProtocolTests.cs

```diff
@@ -410,7 +410,7 @@ public static IEnumerable<string> GetInvalidStatusLine()
             yield return "HTTP/X.Y.Z 200 OK";
             
             // Only pass on .NET Core Windows & SocketsHttpHandler.
-            if (PlatformDetection.IsNetCore && PlatformDetection.IsWindows)
+            if (PlatformDetection.IsNetCore && !PlatformDetection.IsUap && PlatformDetection.IsWindows)
```

Comment:
I was surprised to discover that `PlatformDetection.IsNetCore` returns TRUE on UAP platform as well as non-UAP .NET Core. So, it is needed to add `!PlatformDetection.IsUap` as well.